api/PresentationRequest.json

```diff
@@ -451,7 +451,7 @@
               "version_added": "48"
             },
             "edge": {
-              "version_added": "≤79"
+              "version_added": false
```

Comment:
Is it really correct that Chrome has this but not Edge?

Reply:
https://mdn-bcd-collector.appspot.com/tests/api/PresentationRequest/startWithDevice tests false for me in Chrome as well, so no, it's not correct.

@vinyldarkscratch do you want to back this change out and fix the `startWithDevice` entry as one PR?

Reply:
Good catch!  I created #10493 to take care of all the Chrome data.